### PR TITLE
Enable tests on special GPU device

### PR DESCRIPTION
**Pull request by densmirn**

### CODE REVIEW

**Comment by samir-nasibli** [comment]
Do we really need this resolution for failed tests?
@densmirn provide, please, more detailed description for this case.
It is better create issue ticket for that.

File: tests/conftest.py [hunk 1]
```diff
@@ -49,13 +48,12 @@ def pytest_collection_modifyitems(config, items):
...
 
-    current_queue_is_cpu = dpnp.dpnp_queue_is_cpu()
-    print("")
-    print(f"DPNP: current queue is CPU: {current_queue_is_cpu}")
-    print(f"DPNP: version: {dpnp.version.version}")
-    print("")
-    if not current_queue_is_cpu or os.getenv('DPNP_QUEUE_GPU') == '1':
+    if os.getenv('DPNP_QUEUE_GPU') == '1':
         excluded_tests.extend(get_excluded_tests(test_exclude_file_gpu))
+        # TODO: remove below logic when all the tests passed on special GPU device
```
Comment (samir-nasibli) [inline tests/conftest.py:53]: We do not need special cases.
It is just better add some notes on README.md about this device.

Comment (samir-nasibli) [reply tests/conftest.py:53]: And create issue ticket for that.

**Comment by densmirn** [comment]
`import dpnp` in pytest config makes most tests fail on special GPU device. So dropped usage of dpnp in the config.

**Comment by densmirn** [comment]
> `import dpnp` in pytest config makes most tests fail on special GPU device. So dropped usage of dpnp in the config.

And it's not clear why tests fail.

**Comment by samir-nasibli** [comment]
@densmirn Do we have CI jobs for testing on that "special GPU device"?
If not, then you shouldn't add changes to the working code.

**Pull request closed without merging**